.github/workflows/sample-repo-sync.yml

```diff
@@ -0,0 +1,45 @@
+name: Sample repo sync
+
+on:
+  workflow_dispatch:
+
+permissions:
+  contents: read
+
+jobs:
+  sample-to-repo:
+    name: Update sample in the kalix.io repo
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Checkout JVM SDK
+        # v3.1.0
+        uses: actions/checkout@93ea575cb5d8a053eaa0ac8fa3b40d7e05a33cc8
+        with:
+          path: kalix-jvm-sdk
+          sparse-checkout: |
+            samples/java-spring-choreography-saga-quickstart
+
+      - name: Checkout sample in kalix.io
+        # v3.1.0
+        uses: actions/checkout@93ea575cb5d8a053eaa0ac8fa3b40d7e05a33cc8
+        with:
+          # https://github.com/kalix-io/user-registry-orchestration-saga-workflow
+          repo: kalix-io/user-registry-orchestration-saga-workflow
+          path: user-registry-orchestration-saga-workflow/
```

Comment:
this is wrong, we should target `kalix-io/user-registry-choreography-saga` instead

Reply:
But I have a second thought about this approach. 

If we use such an action, we will need lots of CI configs to cover all the samples that we want to sync. 

Instead, we should have a matrix from source to target repos and a shell script that receiving the pairs and doing all the rest. Then, we only need to add/remove lines from the matrix.

Reply:
Before I matrix, I go flat.